modules/cache/main.tf

```
@@ -56,7 +56,9 @@ resource "aws_s3_bucket_lifecycle_configuration" "build_cache_versioning" {
   rule {
     id     = "AbortIncompleteMultipartUploads"
     status = "Enabled"
-
+    filter {
+      prefix = var.cache_incomplete_lifecycle_prefix
```

suggestion: always set this to an empty string to simplify the code and remove the variable. Or was there a specific reason to add one?

```suggestion
      prefix = ""
```

---

Hello, no specific reason, just for flexibility. 
I agree with you that an empty string is sufficient